src/tools/fuse/rebuild_and_install.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+sudo ./uninstall.sh
+sudo ./build.sh
```

Comment:
```suggestion
./build.sh
```

Does building really need sudo?

Reply:
No - Thanks for pointing that out.